src/main/java/picard/vcf/GenotypeConcordance.java

```diff
@@ -546,19 +692,15 @@ final public static TruthAndCallStates determineState(final VariantContext truth
                 // Truth reference is shorter than call reference
                 final String suffix = getStringSuffix(callRef, truthRef, "Ref alleles mismatch between: " + truthContext + " and " + callContext);
                 truthRef = truthRef + suffix;
-                if (null == truthState) {
-                    truthAllele1 = truthGenotype.getAllele(0).getBaseString() + suffix;
-                    truthAllele2 = truthGenotype.getAllele(1).getBaseString() + suffix;
-                }
+                truthAllele1 = truthGenotype.getAllele(0).getBaseString() + suffix;
+                truthAllele2 = truthGenotype.getAllele(1).getBaseString() + suffix;
```

Comment:
I think the current implementation works for a deletion, but not an insertion.  It should be added in-before the rest of the allele, which for deletions is empty, but for an insertion could be many bases.  I'll see if I can fix it.  Hrmn.